pkgs/by-name/al/alsa-utils/package.nix

```diff
@@ -30,6 +58,7 @@ stdenv.mkDerivation rec {
   postFixup = ''
     mv $out/bin/alsa-info.sh $out/bin/alsa-info
     wrapProgram $out/bin/alsa-info --prefix PATH : "${lib.makeBinPath [ which pciutils ]}"
+    wrapProgram $out/bin/aplay --set-default ALSA_PLUGIN_DIR ${plugin-dir}
```

Comment:
I'm wondering if it wouldn't have been better to do this in a wrapper derivation. Now I have to recompile alsa-utils to have avoid a QT dependency (I have alsa on a headless system with very very small storage).